docs/man_pages/lib-management/kendoui-install.md

```diff
@@ -0,0 +1,51 @@
+kendoui install
+==========
+
+Usage | Synopsis
+------|-------
+List all versions | `$ appbuilder kendoui install [--verified] [--latest]`
```

Comment:
correct concern, but without interactive terminal, the user will not be able to understand the index of the version he needs